client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -228,6 +232,14 @@ public void deploy(String modelId, ActionListener<MLDeployModelResponse> listene
         }));
     }
 
+    @Override
+    public void createConnector(MLCreateConnectorInput mlCreateConnectorInput, ActionListener<MLCreateConnectorResponse> listener) {
+        MLCreateConnectorRequest createConnectorRequest = new MLCreateConnectorRequest(mlCreateConnectorInput);
+        client.execute(MLCreateConnectorAction.INSTANCE, createConnectorRequest, ActionListener.wrap(listener::onResponse, e -> {
+            listener.onFailure(e);
+        }));
```

Comment:
why do we need to `ActionListener.wrap` here

Reply:
To execute the respective action when response is received.  That's how we have it for all the APIs [here](https://github.com/opensearch-project/ml-commons/blob/main/client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java). Let me know if you have any other suggestion.

Reply:
right, but in this case we're just passing responses through directly to the ActionListener parameter - so why not just hand the client `listener` instead of adding this wrap layer. Doesn't really matter. (my suggestion would be
`client.execute(MLCreateConnectorAction.INSTANCE, createConnectorRequest, listener);` )

Reply:
Got it. Pushed the change